src/lib.rs

```diff
@@ -1250,6 +1250,22 @@ impl Log for NopLogger {
     fn flush(&self) {}
 }
 
+impl<T> Log for &'_ T
+where
+    T: ?Sized + Log,
+{
+    fn enabled(&self, metadata: &Metadata) -> bool {
+        (*self).enabled(metadata)
```

Comment:
Normally I'd write something like `(**self).enabled(metadata)` when forwarding implementations on references. I just thought I'd check to make sure this isn't a possible infinite recursion 🙂

Reply:
I'd forgotten the * and I had a warning, then I added it and the warning went away. I could add a test that would otherwise trigger the infinite recursion if you like.

Reply:
The more tests the merrier!